eve/workers/pod-builder/pod.yml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: worker
+spec:
+  containers:
+    - name: worker
+      image: docker-builder
```

Comment:
here you need to use a jinja template to refer to your image. eve will then do some magic to point it to the right image on the registry, so it should be `{{ image: images['docker-builder'] }}`

Reply:
careful, I made a mistake and edited my comment